gatsby/content/projects/2016-02-10-matrix-appservice-gitter.mdx

```diff
@@ -11,7 +11,7 @@ license: Apache-2.0
 repo: https://github.com/matrix-org/matrix-appservice-gitter
 thumbnail: /docs/projects/images/gitter-logo.svg
 bridges: Gitter
-featured: true
+featured: false
```

Comment:
This will hide old bridge, [`gatsby/src/pages/bridges.js#L157`](https://github.com/matrix-org/matrix.org/blob/e8a6aa747836bd11a06d642b1467599d0eb425c1/gatsby/src/pages/bridges.js#L157)